packages/core-data/src/entities.js

```diff
@@ -17,6 +17,12 @@ import { apiFetch, select } from './controls';
 export const DEFAULT_ENTITY_KEY = 'id';
 
 export const defaultEntities = [
+	{
+		label: __( 'Base' ),
+		name: 'base',
+		kind: 'root',
+		baseURL: '',
+	},
```

Comment:
The only think I'm unsure of in this PR is whether we should consider this endpoint as an entity or not? Are we already using this endpoint in Gutenberg? How are we doing so?

The alternative might be a dedicated action/selector on the 'core-data' store. Any thoughts @epiqueras

Reply:
How is this different from the "Site" entity?

Reply:
This is the "index" endpoint. These are alo site settings somehow but the endpoint is separate because this one is I believe authorized for more folks.

Reply:
Maybe we should call it Public Site?

Reply:
I think it makes sense as an entity if it has multiple properties we want to edit and save in unison.

Reply:
Yeah the `index` is completely public.

Reply:
@epiqueras The index returns, among other properties, information including site name, site description, site icon url, and timezone string. These seem like properties we'd want to edit in the future. 

I'm uncertain, however, if that's possible through the current index endpoint.

Reply:
Ideally, we should have a single read/write endpoint for these "site" settings.

Reply:
Aren't all those properties already present in the settings endpoint?

Reply:
I completely agree. We initially attempted to consolidate site icon url alongside the WP "site" settings REST endpoint, rather than through `index`. That endpoint, however, was only accessible to admin and super-admin roles. We want to display the site icon to all roles.

There's been [work](https://github.com/WordPress/gutenberg/pull/19967) and [discussion](https://wordpress.slack.com/archives/C02RQC26G/p1579198597407600) to provide read-only access to some site configuration properties, but [Timothy has said this won't be tackled until WordPress 5.6](https://github.com/WordPress/gutenberg/pull/22952#discussion_r438124552). I'm happy to try other ways of consolidating information if you have suggestions.

Reply:
Ahh, ok. I guess, for now, we can add this new entity, but let's prefix it with `__unstable` so that we can eventually consolidate things.

Reply:
> Ideally, we should have a single read/write endpoint for these "site" settings.

Hoping to get to this in 5.6

Reply:
To clarify, when you say prefix with `__unstable`, do you mean the name like so?

```javascript
	{
		label: __( 'Base' ),
		name: '__unstableBase',
		kind: 'root',
		baseURL: '',
	},

```

Reply:
Yes